crates/ruma-common/tests/identifiers/ui/01-valid-id-macros.rs

```diff
@@ -1,13 +1,15 @@
 fn main() {
-    let _ = ruma_common::device_key_id!("ed25519:JLAFKJWSCS");
-    let _ = ruma_common::event_id!("$39hvsi03hlne:example.com");
-    let _ = ruma_common::event_id!("$acR1l0raoZnm60CBwAVgqbZqoO/mYU81xysh1u7XcJk");
-    let _ = ruma_common::mxc_uri!("mxc://myserver.fish/sdfdsfsdfsdfgsdfsd");
-    let _ = ruma_common::room_alias_id!("#alias:server.tld");
-    let _ = ruma_common::room_id!("!1234567890:matrix.org");
-    let _ = ruma_common::room_version_id!("1");
-    let _ = ruma_common::room_version_id!("1-custom");
-    let _ = ruma_common::server_signing_key_id!("ed25519:Abc_1");
-    let _ = ruma_common::server_name!("myserver.fish");
-    let _ = ruma_common::user_id!("@user:ruma.io");
+    _ = ruma_common::device_key_id!("ed25519:JLAFKJWSCS");
+    _ = ruma_common::event_id!("$39hvsi03hlne:example.com");
+    _ = ruma_common::event_id!("$acR1l0raoZnm60CBwAVgqbZqoO/mYU81xysh1u7XcJk");
+    _ = ruma_common::mxc_uri!("mxc://myserver.fish/sdfdsfsdfsdfgsdfsd");
+    _ = ruma_common::room_alias_id!("#alias:server.tld");
+    _ = ruma_common::room_id!("!1234567890:matrix.org");
+    _ = ruma_common::room_version_id!("1");
+    _ = ruma_common::room_version_id!("1-custom");
+    _ = ruma_common::server_signing_key_id!("ed25519:Abc_1");
+    _ = ruma_common::server_name!("myserver.fish");
+    _ = ruma_common::user_id!("@user:ruma.io");
+
+    _ = ruma_common::owned_user_id!("@user:ruma.io");
```

Comment:
Why test only one?

Reply:
Because I'm lazy and the macros are all the same.. 😅